### ISSUE TITLE
IncompatibleClassChangeError: Found class com.sk89q.worldedit.extent.clipboard.io.ClipboardFormat, but interface was expected

### ISSUE THREAD

**Issue by mibby** [NLthijs48/AreaShop#448]
Paper dev 409 (Spigot 1.13.2)
FAWE-1.13 dev 99
AreaShops dev 265

```
[13:40:39] [Server thread/INFO]: player issued server command: /as unrent
[13:40:39] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'as' in plugin AreaShop v2.5.0#265
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[patched_1.13.2.jar:git-Paper-409]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:150) ~[patched_1.13.2.jar:git-Paper-409]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:729) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1761) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1565) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.2.jar:git-Paper-409]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.2.jar:git-Paper-409]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.2.jar:git-Paper-409]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.IncompatibleClassChangeError: Found class com.sk89q.worldedit.extent.clipboard.io.ClipboardFormat, but interface was expected
	at me.wiefferink.areashop.handlers.WorldEditHandler7.restoreRegionBlocks(WorldEditHandler7.java:68) ~[?:?]
	at me.wiefferink.areashop.regions.GeneralRegion.restoreRegionBlocks(GeneralRegion.java:771) ~[?:?]
	at me.wiefferink.areashop.regions.GeneralRegion.handleSchematicEvent(GeneralRegion.java:1409) ~[?:?]
	at me.wiefferink.areashop.regions.RentRegion.unRent(RentRegion.java:711) ~[?:?]
	at me.wiefferink.areashop.commands.UnrentCommand.execute(UnrentCommand.java:79) ~[?:?]
	at me.wiefferink.areashop.managers.CommandManager.onCommand(CommandManager.java:142) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[patched_1.13.2.jar:git-Paper-409]
	... 15 more
```

**Comment by mibby** [comment]
@NLthijs48 Any possible fix for this issue? :( Can't unrent plots that rollback with a schematic.

**Comment by Sprungente** [comment]
Short other Question back: It FAWE still broken with copy storageitems like chest?

**Comment by mibby** [comment]
@Sprungente Yes. FAWE does not seem to save NBT contents with chests when copy/pasting. However likewise, Vanilla WorldEdit also has this issue and hasn't fixed it for 1.13 yet.

**Comment by NLthijs48** [comment]
I'm working on FAWE support, this issue keeps track of it: https://github.com/NLthijs48/AreaShop/issues/436

I'll close this one as duplicate to keep it organized
